Live/Classes/HUD/HUD.cpp

```diff
@@ -1,6 +1,6 @@
 #include "HUD.h"
 
-HUD::HUD(const Player* player) {
+HUD::HUD(Player* player) {
```

Comment:
Why does HUD need to have knowledge about the player again?

Reply:
To show the player hunger, stamina.